doc/man/scons.xml

```diff
@@ -2981,12 +2981,24 @@ env.Program('bar', source='bar.c foo.c'.split())
 
 <note>
 <para>
-Top-relative paths <emphasis role="bold">only</emphasis> work
-where &SCons; will interpret the path (see examples).
-To be used in other contexts the string must first be converted
+Top-relative paths only work
+for objects that are participants in the build
+(sources and targets).
+To be used in other contexts
```

Comment:
Invert this sentence?
The string must first be converted to a relative or absolute path to be used in other contexts... 

I think it reads a bit better?

Reply:
Shrug. Done.